### PR TITLE
OH2-304 | Implement disease create and update

### PR DESCRIPTION
**Pull request by SteveGT96**
See OH2-304.

### CODE REVIEW

**Comment by mwithi** [comment]
@SteveGT96 please resolve conflicts (I think it is the topic suggested by @gasp on [Slack](https://join.slack.com/t/openhospitalworkspace/shared_invite/zt-cg9iu1d0-sjTuWWziFgcDqiTWyMJBAw))

**Comment by SteveGT96** [comment]
Hello @gasp. Please, could you also review this ?

**Comment by mwithi** [comment]
@SteveGT96 conflicts again...

**Review by SilverD3 — APPROVED** [review]
The code looks clean and it's working

**Comment by mwithi** [comment]
> The code looks clean and it's working

ok, thanks, let's wait for @gasp review

File: src/mockServer/routes/diseases.js
Comment (gasp) [inline src/mockServer/routes/diseases.js]: Nitpick: What about filtering here to have more realistic results (and easier to test and debug)
```suggestion
    server.get("/opd").intercept((req, res) => {
      res.status(200).json(diseasesDTO.filter(({opdInclude}) => opdInclude);
    });
    server.get("/ipd/in").intercept((req, res) => {
      res.status(200).json(diseasesDTO.filter(({ipdInInclude}) => ipdInInclude));
    });
    server.get("/ipd/out").intercept((req, res) => {
      res.status(200).json(diseasesDTO.filter(({ipdOutInclude}) => ipdOutInclude));
    });
```

File: src/components/accessories/admin/wards/editWard/EditWard.tsx
Comment (gasp) [inline src/components/accessories/admin/wards/editWard/EditWard.tsx:26]: That's great, thanks!

File: src/routes/Admin/AdminRoutes.tsx
Comment (gasp) [inline src/routes/Admin/AdminRoutes.tsx]: this could use PATHS const.
(not mandatory)

File: src/state/diseases/actions.ts
Comment (gasp) [inline src/state/diseases/actions.ts:30]: nice catch!

**Review by gasp — APPROVED** [review]
All good, I added minor comments, but that's good enough to be merged!

File: src/mockServer/routes/diseases.js
Comment (SteveGT96) [reply src/mockServer/routes/diseases.js]: Looks  better.

**Review by gasp — APPROVED** [review]

**Comment by mwithi** [comment]
Please, @SteveGT96 resolve the conflicts

**Review by mwithi — APPROVED** [review]
Approved by @gasp

**Comment by mwithi** [comment]
After merged this, the admin route goes straight to admin/wards and results in a 404

image

**Pull request merged**
